README.md

```diff
@@ -1010,6 +1010,17 @@ skate.prop.boolean({
 
 Generally built-in properties will only return a definition containing `coerce`, `deserialize` and `serialize` options. They may also define a `deafult`, such as with the `boolean` property.
 
+Be aware that if you pass options into a property that you might be overriding default behaviour. With this in mind, it is up to you to make sure you callback or properly set anything that you override:
+
+```js
+const slot = skate.prop.slot();
+const prop = Object.assign(slot, {
```

Comment:
And it's just JS. I don't think it's worthwhile to document the implementation details for each built-in. In practice, we've found overriding to be an edge case and I think is something that we'd likely deprecate in the future.